src/up.js

```diff
@@ -92,6 +92,12 @@ export default function up() {
           type: 'string',
         })
         .group(['url', 'livereload', 'no-livereload', 'open', 'no-open', 'print-index', 'cache'], 'AEM Options')
+        .option('allow-unauthorized', {
+          alias: 'allowUnauthorized',
+          describe: 'Whether to allow unauthorized access to server',
```

Comment:
maybe this is not a very good description... I think `allowInsecure` would be better